demo/src/App.js

```diff
@@ -164,8 +157,7 @@ export default class App extends PureComponent {
             sliderValues,
             colorValues,
```

Comment:
Maybe this is a general thing we should discuss tomorrow, but as soon as the order of these goes out of wack (i.e the keys of `sliderValues` does not match the order of the keys of `colorValues`), the sliders correspond to the wrong channels.  I think we need to enforce a `sort` call somewhere or something (maybe when the `MicroscopyViewerLayer` initializes): https://stackoverflow.com/questions/5467129/sort-javascript-object-by-key. At the minimum, I think we should understand why this works the way it does (there is in fact a sort call in `MicroscopyViewerLayerBase`).

Reply:
What ever order the keys are provided in the config is what is respected in the rest of the application, to my knowledge.

Reply:
I need to look at `colorValues` and `slider` values, but the logic in the `tiff_utils` was necessary because we can iterate over the object as its provided.